CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ The changelog for `MessageKit`. Also see the [releases](https://github.com/Messa
 
 ### Added
 
+- **Breaking Change** Added typing indicator support to MessageKit by adding an extra section to `MessagesCollectionView` when needed. This included adding the following views: `BubbleCircle`, `TypingBubble`, `TypingIndicator`, `TypingBubbleCell`. Determine if the section is reserved for the typing indicator with `isSectionReservedForTypingIndicator(_ section: Int) -> Bool`, and set the typing indicators visiability with `setTypingIndicatorHidden(_ isHidden: Bool, animated: Bool, whilePerforming updates: (() -> Void)? = nil, completion: ((Bool) -> Void)?=nil)`
```

Comment:
This is more a reminder to myself to improve this CHANGELOG entry before merging

Reply:
Yes, this is a valid concern that I have as well. I like the general idea but I haven't gone deep into how this will affect MessageKit